src/lxc/start.c

```diff
@@ -962,33 +962,6 @@ static int do_start(void *data)
 		goto out_warn_father;
 	}
 
-	/* The container has been setup. We can now switch to an unprivileged
```

Comment:
I see no reason not to do this, but the changelog doesn't tell me why you *want* to do this.  Is there sometimes a problem opening devnull?  Is it just for the principle of the thing?  On failure this task will just go away, right, so there's no cleanup reason to worry about?

Reply:
This is just on principle. In case someone wants to add an operation later on that requires privilege it should not be blocked by switching IDs.